uwblueprint/sistema
# Start calendar page

# Notion ticket link

[Implement React Calendar Component](https://www.notion.so/uwblueprintexecs/Implement-React-Calendar-Component-f5f84b96f5b3498c9bf94ff262bb0ea3)


## Implementation description

* add unstyled calendar component from react-calendar library
* add dummy code for props that may be useful for future changes (`onClickDay` and `tileContent`)


## Steps to test

1. go to localhost:3000/calendar and verify a calendar appears:
![image](https://github.com/uwblueprint/sistema/assets/91142355/079117f7-fb75-4b56-a476-32e32f342ad6)


## What should reviewers focus on?

* extra props? any opinions

## Checklist
